EU4toV2/Source/EU4World/Regions/Areas.cpp

```diff
@@ -12,9 +12,9 @@ EU4::Areas::Areas(std::istream& theStream)
 
 void EU4::Areas::registerKeys()
 {
-	registerRegex(R"([\w_]+)", [this](const std::string& areaName, std::istream& theStream) {
+	registerRegex(commonItems::stringRegex, [this](const std::string& areaName, std::istream& theStream) {
```

Comment:
unless stringregex includes underscores, this isn't a replacement.

Reply:
Doesn't it? Most of the regex is about what's **not** included. The test are green.